.github/workflows/main.yml

```diff
@@ -227,7 +227,7 @@ jobs:
         run: |
           MATRIX="matrix={\"include\":["
           # Regular tests: Run different test suites in parallel with specific features
-          MATRIX="${MATRIX}{\"args-tests\":\"threshold -- --skip centralized --skip full_gen_tests\"},{\"args-tests\":\"centralized -- --skip threshold --skip full_gen_tests\"}"
+          MATRIX="${MATRIX}{\"args-tests\":\"threshold -- --skip centralized --skip full_gen_tests --skip nightly_tests\"},{\"args-tests\":\"centralized -- --skip threshold --skip full_gen_tests\"}"
```

Comment:
Maybe just set the skip to `nightly`

Reply:
Yes, yes, what I mean is to write
```
          MATRIX="${MATRIX}{\"args-tests\":\"threshold -- --skip centralized --skip full_gen_tests --skip nightly\"},{\"args-tests\":\"centralized -- --skip threshold --skip full_gen_tests\"}"
```
I.e. without the `_tests` suffix on `nightly` :)